k8s/helm-chart/dynatrace-gcp-function/values.yaml

```diff
@@ -27,11 +27,22 @@ deploymentType: "all"
 dynatraceAccessKey: ""
 
 # LOGS VALUES - REQUIRED for 'logs' and 'all' deployment
-# ActiveGate endpoint used to ingest logs to Dynatrace, for example: https://environemnt-active-gate-url:9999/e/environment-id"
-dynatraceLogIngestUrl: ""
+
 # subscription id of log sink pubsub subscription
 logsSubscriptionId: ""
 
+#dynatraceLogIngestUrl ActiveGate endpoint used to ingest logs to Dynatrace, for example: https://environemnt-active-gate-url:9999/e/environment-id"
+#used ONLY when the activeGate.autoDeploy is false
+dynatraceLogIngestUrl: ""
+
+activeGate:
+  #autoDeploy if set to true the Active Gate will be deployed in k8s. In that case also the activeGate.dynatracePaasToken must be filled.
+  autoDeploy: true
+  #autoDeployPort port used by Active Gate pod on which AG is exposed.
+  autoDeployPort : 9999
```

Comment:
Do we have a specific use-case for k8s service custom port? This `values.yaml` file keeps growing and feedback from customers have been that they feel overwhelmed by amount of options. I'd remove `autoDeployPort` if it's not absolutely necessary.

Reply:
Removed it.